src/python/setup.py

```diff
@@ -179,11 +179,12 @@ def do_not_install(require):
     ):
         install_requires.append("llvmlite == 0.31.0")
 
-    if sys.version_info[0] == 3 and sys.version_info[1] == 8:
```

Comment:
I would much prefer a change like the following:
```python
    if sys.version_info[0] == 3 and sys.version_info[1] == 8:
         # Only 4.0 betas support Python 3.8
         install_requires.append("coremltools==4.0b3; python_version >= 3.8.0',")
     else:
         install_requires.append("coremltools==3.3; python_version < 3.8.0,")
```
I think that is much more readable and accomplishes the same thing.

Reply:
Sure thing!

Reply:
Changed.